plugins/nf-azure/src/main/nextflow/cloud/azure/batch/AzBatchTaskHandler.groovy

```diff
@@ -120,6 +122,11 @@ class AzBatchTaskHandler extends TaskHandler {
             task.stdout = outputFile
             task.stderr = errorFile
             status = TaskStatus.COMPLETED
+            TaskExecutionInformation info = taskExecutionInfo(taskKey)
+            if (info.result() == TaskExecutionResult.FAILURE) {
+                log.error("[AZURE BATCH] Task failed. ERROR code: ${info.failureInfo().code()}; message: ${info.failureInfo().message()}.")
```

Comment:
Likely this is not needed, otherwise the error is reported twice

Reply:
You need to check if the result is a FAILURE before accessing the failureInfo.

Reply:
I can of course remove the log.error statement. Is that what you meant?